ast/src/parsed/display.rs

```diff
@@ -119,7 +144,7 @@ impl<T: Display> Display for MachineStatement<T> {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result {
         match self {
             MachineStatement::Degree(_, degree) => write!(f, "degree {};", degree),
-            MachineStatement::Pil(_, statement) => write!(f, "{statement};"),
+            MachineStatement::Pil(_, statement) => write!(f, "{statement}"),
```

Comment:
Did this produce a duplicate `;` before?

Reply:
yep